app/src/main/java/agency/five/codebase/android/movieapp/data/di/DataModule.kt

```diff
@@ -0,0 +1,12 @@
+package agency.five.codebase.android.movieapp.data.di
+
+import agency.five.codebase.android.movieapp.data.repository.FakeMovieRepository
+import agency.five.codebase.android.movieapp.data.repository.MovieRepository
+import kotlinx.coroutines.Dispatchers
+import org.koin.dsl.module
+
+val dataModule = module {
+    single<MovieRepository> {
+        FakeMovieRepository(ioDispatcher = Dispatchers.IO)
+    }
+}
```

Comment:
All files should have an empty line at the end. Android Studio can add it automatically by checking
Preferences -> Editor -> General -> Ensure line feed at file end on Save